### DISCUSSION TITLE
is version 3 available for download from github instead of the webservice?

### DISCUSSION THREAD

**Discussion by GZLiew** [Dav1dde/glad#400] [General]
Hi I notice when I landed on this repo it points to the branch v2 of glad
may I know if there's a v3?
I want be able to use fetchContent in my CMakeList to grab GLAD library 

Thanks

**Comment by GZLiew** [comment]
thats ok it appears that what Im actually looking at is glad 0.1.36 instead of 2

**Reply by Dav1dde** [reply]
glad 0.1.36 is the master branch.

You're fine to use glad2 or glad1 (0.1.36), I'll support both versions, glad2 is just more maintainable for me and has a few more options
